README.md

```diff
@@ -1,18 +1,40 @@
 # Kotlin Android Architecture Library - Kaal - by eMan
 
+[![Latest version](https://img.shields.io/github/v/release/eManPrague/kaal)](https://github.com/eManPrague/kaal/releases/tag/v0.8.0)
+
 [![Slack channel](https://img.shields.io/badge/Chat-Slack-blue.svg)](https://kotlinlang.slack.com/messages/kaal/)
 
 ### Usage
 
-All artifacts are available and distributed by using the `jcenter()`
+:warning: The artifacts was moved from jCenter.
 
-You should you artifact which you need. E.g. a domain artifact you will use in your domain layer, 
-but you can use it also in data and infrastructure, because you need e.g. instance to the `Result` class.
+All artifacts are available and distributed by using the [eMan Nexus](https://nexus.eman.cz/service/rest/repository/browse/maven-public/) repository.
```

Comment:
`by using the` je zvláštní kombinace. Stašilo by `using the`.

Reply:
Já jsem ten začátek nepsal, jen jsem měl měnil ten repozitář, ale zkontroluju to.